### DISCUSSION TITLE
Snooze alarm from watch not working

### DISCUSSION THREAD

**Discussion by kameamea** [NightscoutFoundation/xDrip#3501] [Q&A]
Snoozing alarm from my Samsung watch 4 used to work fine. After switching to a new phone, Huawei p30, it doesn't work anymore. It used to work on the same phone model last year however. So maybe i missed some setting necessary for this? 
I assigned the home button to run snooze alert when double pressed, and i also see the message "sending snooze"on the watch when i do so. But the phone just doesn't stop it's alarm. Any ideas?

**Comment by SimonPhilpott** [comment]
You might need to enable toasts in the Xdrip settings on your watch.

**Comment by TheFlyingDave** [comment]
Same behavior here. I think this is a recent issue - it worked perfectly for me for many months (possibly years). Suddenly it stopped working. The watch says "Sending Snooze" but this doesn't seem to be received by the phone. I have toasts enabled. The only change is I recently updated the xDrip software on my watch by uninstalling/reinstalling it.

**Reply by TheFlyingDave** [reply]
I should add that I'm also using a Samsung watch 4. Phone is a Samsung Galaxy S20fe. I get the same behavior if I snooze by double-pressing the side button (assigned to snooze) or running the xDrip Snooze Alert app manually.

**Reply by TheFlyingDave** [reply]
NVM. Rebooting both watch and phone resolve the situation. I can once again snooze alerts from my watch (yay).
